packages/vuetify/src/components/VExpansionPanel/VExpansionPanel.tsx

```diff
@@ -38,16 +38,20 @@ export const VExpansionPanel = defineComponent({
     const { roundedClasses } = useRounded(props)
     const isDisabled = computed(() => groupItem?.disabled.value || props.disabled)
 
+    const selectedIndices = computed(() => groupItem.group.items.value.reduce<number[]>((arr, item, index) => {
+      return groupItem.group.selected.value.includes(item.id) ? [...arr, index] : arr
+    }, []))
```

Comment:
Spread in reduce is slow as hell
```suggestion
    const selectedIndices = computed(() => groupItem.group.items.value.reduce<number[]>((arr, item, index) => {
      if (groupItem.group.selected.value.includes(item.id)) arr.push(index)
      return arr
    }, []))
```